packages/commonwealth/client/scripts/views/components/proposals/voting_results.tsx

```diff
@@ -38,6 +55,14 @@ export const VotingResults = (props: VotingResultsProps) => {
     };
   }, [forceRerender]);
 
+  const { data } = useAaveProposalVotesQuery({
+    moduleReady: app.chain?.network === ChainNetwork.Aave || !isLoading,
+    chainId: app.chain?.id,
+    proposalId: proposal.identifier,
+  });
+
+  const votes = data || proposal.getVotes();
```

Comment:
The data in `const votes = data || proposal.getVotes();` will be nullish until its fetched, just to confirm if `proposal.getVotes()` returns a valid array/obj?

Reply:
`proposal.getVotes()` returns `Object.values(...)` so it will return a valid array even if there are no votes.